app/apollo/models/channel.schema.js

```diff
@@ -57,9 +57,6 @@ const ChannelSchema = new mongoose.Schema({
       description: {
         type: String,
       },
-      location: {
```

Comment:
If location is actually removed from the schema, will that affect the ability to make a graphql query to get 'location'?  Code such as the CLI refers to location, even if the value is not used, e.g. https://github.ibm.com/alchemy-containers/armada-cli/blob/master/plugin/models/razee/configuration.go#L30
If this change would cause queries to fail, it breaks backwards compatibility even if the location attribute is essentially 'meaningless'.  It would be safer to keep it in the schema I would think... but if we think it's better to remove it then we need to thoroughly test in Stage, including with the existing CLI, to confirm no breakage.

Reply:
mongo location field is not used anymore.

Reply:
tested again - location field is still returned:
{
  "data": {
    "channels": [
      {
        "uuid": "3b49a7b4-b92f-45f5-80a1-02611ec26e4b",
        "orgId": "851d4530-75dc-4a60-9fc0-e6240316b2de",
        "name": "service-team-channel",
        "created": "2021-06-04T15:07:40.853Z",
        "data_location": "wdc",
        "versions": [
          {
            "name": "test-errors",
            "description": null,
            "uuid": "c5b53b04-1dae-4a9f-b850-a0f7f76b94bd",
            "created": "2021-06-04T15:09:02.307Z",
            "location": null
          }
        ]
      }
    ]
  }
}